### PR TITLE
Adding new changes to your gitfile.txt

### PR DESCRIPTION
**Pull request by KaranChaudhary12**
Added 2 lines in gitfile.txt.
Firstly changed test1.txt to something else by mistake, then changed it into the previous version.

### CODE REVIEW

File: gitfile.txt [hunk 1]
```diff
@@ -1,4 +1 @@
-Hello!
```
Comment (SaumalyaSen) [inline gitfile.txt:1]: add this line

File: gitfile.txt [hunk 2]
```diff
@@ -1,4 +1 @@
...
-I am Saumalya
```
Comment (SaumalyaSen) [inline gitfile.txt:2]: add this line

File: gitfile.txt [hunk 3]
```diff
@@ -1,4 +1 @@
...
-I work for LTIMindtree
-Currently, I am studying GIT & GITHUB
```
Comment (SaumalyaSen) [inline gitfile.txt:4]: add this one also

**Review by SaumalyaSen — CHANGES_REQUESTED** [review]
Please check and change

File: gitfile.txt (hunk 2 above)
Comment (KaranChaudhary12) [reply gitfile.txt:2]: added.

File: gitfile.txt (hunk 3 above)
Comment (KaranChaudhary12) [reply gitfile.txt:4]: added

**Review by KaranChaudhary12 — COMMENTED** [review]
ALL CHANGE IS ACCEPTED BY KARAN.

**Review by SaumalyaSen — APPROVED** [review]
Approved!

**Pull request merged**